app/src/main/java/org/mifos/mobile/ui/fragments/SavingsAccountTransactionDetailFragment.java

```diff
@@ -0,0 +1,202 @@
+package org.mifos.mobile.ui.fragments;
```

Comment:
we should be able to combine these 3 fragments into a single fragment and maintain the transaction type (savings, loan, client). Currently, most of the code is repeated in these 3 fragments.

Reply:
> we should be able to combine these 3 fragments into a single fragment and maintain the transaction type (savings, loan, client). Currently, most of the code is repeated in the 3 transaction detail fragments.

thanks @naman14 , my bad, my code just increases the boilerplate. I will update the PR